From 92e8a387bacf05438453cdef4c0633e67978cf9f Mon Sep 17 00:00:00 2001
From: Star Automation <star@marvell.com>
Date: Tue, 12 Feb 2013 09:42:11 +0200
Subject: [PATCH 513/609] Align: Add version.txt files needed to imcrement
 version number Signed-off-by: Star Automation
 <star@marvell.com>

Signed-off-by: Seif Mazareeb <seif@marvell.com>
---
 .../mach-armada370/armada_370_family/version.txt   |    1 +
 .../arm/mach-armadaxp/armada_xp_family/version.txt |    1 +
 arch/arm/plat-armada/common/version.txt            |    1 +
 arch/arm/plat-armada/mv_hal/version.txt            |    1 +
 4 files changed, 4 insertions(+)
 create mode 100644 arch/arm/mach-armada370/armada_370_family/version.txt
 create mode 100644 arch/arm/mach-armadaxp/armada_xp_family/version.txt
 create mode 100644 arch/arm/plat-armada/common/version.txt
 create mode 100644 arch/arm/plat-armada/mv_hal/version.txt

--- /dev/null
+++ b/arch/arm/mach-armada370/armada_370_family/version.txt
@@ -0,0 +1 @@
+2013_Q1.0
--- /dev/null
+++ b/arch/arm/mach-armadaxp/armada_xp_family/version.txt
@@ -0,0 +1 @@
+2013_Q1.0
--- /dev/null
+++ b/arch/arm/plat-armada/common/version.txt
@@ -0,0 +1 @@
+2013_Q1.0
--- /dev/null
+++ b/arch/arm/plat-armada/mv_hal/version.txt
@@ -0,0 +1 @@
+2013_Q1.0
